float64 a
float64 b
float64 c
---
float64 set_a
float64 set_b
float64 set_c
